operations/network/management_network/apply_custom_config_1.0.md

```diff
@@ -6,13 +6,13 @@ Apply the backed up site connection configuration with a couple modifications. S
 
 * Access to the switches
 * Custom switch configurations
-  * [Backup Custom Configurations](backup_custom_config.md)
+  * [Backup Custom Configurations](backup_custom_configurations.md)
 * Generated switch configurations already applied
-  * [Apply Switch Configurations](apply_switch_configs.md)
+  * [Apply Switch Configurations](apply_switch_configurations.md)
 
 ## Aruba
```

Comment:
What is the purpose of this section? The heading of just Aruba does have a delightful air of mystery, but while I do find that appealing on a personal level, I think maybe we should make the heading a little more informative.